runtime/mainnet/src/lib.rs

```diff
@@ -1040,4 +1132,198 @@ mod tests {
 
 		assert_eq!(deposit(2, 64), system_para_deposit(2, 64))
 	}
+
+	#[test]
+	fn treasury_account_is_pallet_id_truncated() {
+		assert_eq!(TreasuryAccount::get(), TREASURY_PALLET_ID.into_account_truncating());
+	}
+
+	const ALICE: [u8; 32] = [1; 32];
```

Comment:
Using the predefined dev account constants from sp-keyring saves recreating the wheel. It's an additional dev dependency import, but brings consistency and clarity.

This was done across the SDK codebase in latest stable.

Reply:
habits :P

Reply:
https://github.com/r0gue-io/pop-node/pull/428/commits/6d078eb3e6c15af2facb1aca85d5dd8e62694be1

Reply:
We did the same, only realised the usefulness of sp-keyring thanks to SDK and it allowed the removal of a lot of unnecessary code.